plugins/fs/guest-js/index.ts

```diff
@@ -1322,6 +1322,28 @@ async function watchImmediate(
   }
 }
 
+/**
+ * Get the size of a file or directory.
```

Comment:
```suggestion
 * Get the size of a file or directory. For files, the `stat` functions can be used as well.
 *
 * If `path` is a directory, this function will recursively iterate over every file and every directory inside of `path` and therefore will be very time consuming if used on larger directories.
```

Maybe we should also add a note on the stat functions that it does not return the expected dir size and that this function should be used if that is required.